src/common/utils/modulesRegistry.h

```diff
@@ -2,10 +2,21 @@
 
 #include "registry.h"
 
+#include <common/utils/json.h>
+
 #include <filesystem>
 
 namespace fs = std::filesystem;
 
+namespace NonLocalizable
+{
+    const static wchar_t* MONACO_LANGUAGES_FILE_NAME = L"modules\\FileExplorerPreview\\languages.json";
```

Comment:
Can we rename to something like `monaco_languages.json`. That would be more unique and clear.

Reply:
I've added this to #15726 and I'll fix it for 0.57